### ISSUE TITLE
Drop down?

### ISSUE THREAD

**Issue by hellomiakoda** [Guake/guake#701]
It's said to be a drop down terminal, but it doesn't drop down. It fades in. Looks kinda silly. My old drop down terminal actually slid down reminiscent of the notification tray in Android. I wish Guake did that.

**Comment by Stibbons** [comment]
indeed, it's a fade-in terminal :) But the animation is actually done by your window manager, we don't do any animation manually.

**Comment by Tymek** [comment]
This can be done with Animation Tweaks extension. https://extensions.gnome.org/extension/1680/animation-tweaks/

**Comment by PhilippWoelfel** [comment]
> This can be done with Animation Tweaks extension. https://extensions.gnome.org/extension/1680/animation-tweaks/

With that extension, I can only get the drop down animation for "open" to work, not for "close". Close is always a fade.

**Comment by plsdosomething** [comment]
> This can be done with Animation Tweaks extension. https://extensions.gnome.org/extension/1680/animation-tweaks/

@Tymek Thank you for the suggestion. Now it slides down (or rotates down if I want to) when I open guake, but the closing animation stays the same not matter what option I use. What did you do that it slides up when closing the window?
I use gnome keyboard shortcut that runs the command guake-toggle. It seems strange to me, that the appearing animation works like I want it to, but the disappearing animation stays the same "fade out" animation

**Comment by Optiligence** [comment]
@plsdosomething for me it appears that the global toggles »… Effects [ for windows ]« on the Open and Close tab have to be enabled for the profile options to work.

**Comment by rgpublic** [comment]
What's a bit sad is that Yakuake, a KDE app, has that effect out-of-the-box even when run under Gnome. I think it's an integral part of why I'm using such a terminal in the first place. There are numerous other non-slide-down terminals. That's why I think this effect should be built-in (just like in Yakuake) and not rely on the external window manager.

**Comment by gsemet** [comment]
Hello. Feel free to provide your merge request to implement this feature !

**Comment by paulo-roger** [comment]
Animation Tweaks is abandoned :(

**Comment by karimshalapy** [comment]
I would love to have control over this.

**Comment by XargonWan** [comment]
Following this issue as well

**Comment by sswastik02** [comment]
Hi @gsemet 
If possible can you please specify an area in the codebase where you think the fix might be implemented, It may serve as a good starting point for us all.

Thank you in advance.

**Comment by Davidy22** [comment]
It's more like a feature addition than a fix. It would be alterations to the show() and hide() methods in the source to maybe do some window size fiddling. Need to take care to preserve priot window height, split screen proportions, scroll positions, etc. Alternately, you can make the window animation the job of the compositor.

**Comment by kamikaze** [comment]
what a shame. don't call this app as *uake since it behave in a wrong way

**Comment by Zeratoxx** [comment]
I kind of got the drop down effect with the extension ["Burn my Windows"](https://extensions.gnome.org/extension/4679/burn-my-windows/) and its "Glide" effect with these parameters: 

> ***(Loosely translated from German, probably named different in reality)***
> 
> Duration of animation [ms]: 124
> Size: 1.00
> Compress: 0.00
> Tilt: -0.15
> Displacement: 0.50
